api/client/samples/similar_regions/similar_region.py

```diff
@@ -350,7 +350,7 @@ def _load_property_from_file(self, prop_name, path_prefix):
         # will return array of long-term averages for each time interval within a year, even for static
         result = np.zeros(self.t_int_per_year)
         try:
-            raw_data = pd.read_csv(full_path, parse_dates=[0,1], infer_datetime_format=True)
+            raw_data = pd.read_csv(full_path, parse_dates=[0,1], infer_datetime_format=True).dropna()
```

Comment:
I think it's nice to be explicit w/ data cleaning operations that are happening automatically - for instance, in this case, I think it would be nice to print a message that N rows are being dropped due to containing nulls.

Something like this:

```python
            raw_data = pd.read_csv(full_path, parse_dates=[0,1], infer_datetime_format=True)
            num_rows_with_na = raw_data.isna().sum().max()
            if num_rows_with_na:
                self._logger.warn(f"Note: dropping {num_rows_with_na} rows that contain null values.")
                raw_data.dropna(inplace=True)
```

WDYT?

Reply:
Rather significant amount of missing data can be tolerated (as long as there is at least one valid data point per interval), but sure, it does not hurt, I guess... Done in the next commit.